### PR TITLE
fix(log): Trace parameters to align with profile

### PR DESCRIPTION
**Pull request by epage**
This is a follow up to #13532 and prep for #13399

### CODE REVIEW

File: src/cargo/core/compiler/fingerprint/mod.rs [hunk 1]
```diff
@@ -399,7 +399,10 @@ pub use dirty_reason::DirtyReason;
 /// transitively propagate throughout the dependency graph, it only forces this
 /// one unit which is very unlikely to be what you want unless you're
 /// exclusively talking about top-level units.
-#[tracing::instrument(skip(build_runner, unit))]
+#[tracing::instrument(
+    skip(build_runner, unit),
+    fields(package_id = unit.pkg.package_id().to_string(), target = unit.target.name())
```
Comment (weihanglo) [inline src/cargo/core/compiler/fingerprint/mod.rs:404]:
```suggestion
    fields(package_id = %unit.pkg.package_id(), target = unit.target.name())
```

File: src/cargo/core/global_cache_tracker.rs [hunk 2]
```diff
@@ -697,7 +697,7 @@ impl GlobalCacheTracker {
     ///
     ///    These orphaned files will be added to `delete_paths` so that the
     ///    caller can delete them.
-    #[tracing::instrument(skip_all)]
+    #[tracing::instrument(skip(conn, gctx, base))]
```
Comment (weihanglo) [inline src/cargo/core/global_cache_tracker.rs:700]: Don't quite understand why recording `delete_paths` and `sync_size` here. For `delete_paths` it is used for collecting results and likely to be empty.

```suggestion
    #[tracing::instrument(skip(conn, gctx, base))]
```

Comment (epage) [reply src/cargo/core/global_cache_tracker.rs]: Remove `delete_paths`.  I included `sync_sync` from a debugging perspective to know the context of what is happening.

**Review by weihanglo — APPROVED** [review]

**Comment by weihanglo** [comment]
Thanks!

@bors r+

**Pull request merged**